test/kafo/hook_context_test.rb

```diff
@@ -11,6 +11,24 @@ module Kafo
       specify { context.respond_to?(:param).must_equal true }
       specify { context.respond_to?(:add_module).must_equal true }
       specify { context.respond_to?(:module_enabled?).must_equal true }
+      specify { context.respond_to?(:exit).must_equal true }
+      specify { context.respond_to?(:get_custom_config).must_equal true }
+      specify { context.respond_to?(:store_custom_config).must_equal true }
+      specify { context.respond_to?(:scenario_path).must_equal true }
+      specify { context.respond_to?(:scenario_data).must_equal true }
+    end
+
+    describe "#scenario_data" do
+      specify do
+        Tempfile.open('scenario') do |file|
+          file.write(YAML.dump({foo: 'bar'}))
```

Comment:
this wouldn't work with Ruby 1.8 which is still used with puppet 3.8, would you mind using hash rocket `=>` syntax here please?

Reply:
Updated